website/docs/basics/blocks-intro.md

```diff
@@ -47,8 +59,12 @@ Yes, we do. All block variations are located in the `src/Blocks/variations` fold
 
 ### How do you use components in blocks?
```

Comment:
```suggestion
### How do I use components in blocks?
```

(all other questions are written from perspective of user so might as well this one)